File: VB/UnboundListView.Module/Duplicate.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace UnboundListView.Module
	<NonPersistent> _
	Public Class Duplicate
		Inherits XPObject
		Private Shared myOid As Integer = 0

		Public Sub New(ByVal session As Session)
			MyBase.New(session)
			myOid += 1
			Oid = myOid
		End Sub
		Public Property Title() As String
			Get
				Return GetPropertyValue(Of String)("Title")
			End Get
			Set(ByVal value As String)
				SetPropertyValue(Of String)("Title", value)
			End Set
		End Property
		Public Property Count() As Integer
			Get
				Return GetPropertyValue(Of Integer)("Count")
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue(Of Integer)("Count", value)
			End Set
		End Property
	End Class
End Namespace

File: CS/UnboundListView.Module/Duplicate.cs
using System;
using System.ComponentModel;

using DevExpress.Xpo;
using DevExpress.Data.Filtering;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace UnboundListView.Module {
    [NonPersistent]
    public class Duplicate : XPObject {
        private static int myOid = 0;

        public Duplicate(Session session)
            : base(session) {
            myOid++;
            Oid = myOid;
        }
        public string Title {
            get { return GetPropertyValue<string>("Title"); }
            set { SetPropertyValue<string>("Title", value); }
        }
        public int Count {
            get { return GetPropertyValue<int>("Count"); }
            set { SetPropertyValue<int>("Count", value); }
        }
    }
}
